.github/actions/spelling/excludes.txt

```diff
@@ -1,41 +1,71 @@
 # See https://github.com/check-spelling/check-spelling/wiki/Configuration-Examples:-excludes
 (?:^|/)(?i)COPYRIGHT
 (?:^|/)(?i)LICEN[CS]E
+(?:^|/)(?i)Makfile
+(?:^|/)AndroidManifest\.xml$
+(?:^|/)go\.sum$
 (?:^|/)package(?:-lock|)\.json$
+(?:^|/)perfect/
+(?:^|/)strings\.xml$
 (?:^|/)vendor/
+colors\.dart$
+CPPLINT\.cfg$
+feedback_localization\.dart$
 ignore$
+language_info\.dart$
+release\.sh$
+zh_CN
+zobrist\.dart$
+\.a$
+\.ai$
+\.arb$
 \.avi$
+\.bmp$
+\.bz2$
+\.clang-format$
+\.crt$
+\.dll$
+\.docx?$
+\.drawio$
+\.DS_Store$
+\.eot$
+\.exe$
+\.gif$
+\.gitattributes$
+\.gradle$
+\.graffle$
+\.gz$
+\.icns$
 \.ico$
+\.jar$
+\.jks$
 \.jpe?g$
+\.key$
+\.lib$
 \.lock$
 \.map$
-\.min\.
+\.min\..
 \.mod$
 \.mp[34]$
+\.o$
+\.ocf$
+\.otf$
+\.pdf$
+\.pem$
 \.png$
-\.wav$
-^\.github/
-
```

Comment:
check-spelling tends to automatically sort this file, so blank lines and comments don't really survive.

That said, if you desire, I can manually adjust this PR to fit your house style...